lib/src/sdk.ts

```diff
@@ -98,18 +89,22 @@ enum FeeStrategyEnum {
   CUSTOM = "CUSTOM",
 }
 
-// Should be available from the WalletAPI (zod :( )
-const ExchangeType = {
+export const ExchangeType = {
   FUND: "FUND",
   SELL: "SELL",
   SWAP: "SWAP",
 } as const;
 
+export enum ErrorOrigin {
```

Comment:
Why don't reuse `ExchangeType`? It seems they both serve the same purpose (characterize exchange transaction).